proprietary-files.txt

```diff
@@ -269,7 +269,8 @@ vendor/lib64/vendor.qti.esepowermanager@1.0.so
 vendor/bin/loc_launcher
 vendor/bin/lowi-server
 vendor/bin/xtra-daemon
-vendor/lib64/hw/vendor.qti.gnss@3.0-impl.so
+vendor/etc/vintf/manifest/android.hardware.gnss@2.1-service-qti.xml
```

Comment:
Before manifest we should use (-) dash. Otherwise,
`build/make/core/Makefile:49: error: VINTF metadata found in PRODUCT_COPY_FILES: vendor/xiaomi/sdm710-common/proprietary/vendor/etc/vintf/manifest/android.hardware.gnss@2.1-service-qti.xml:vendor/etc/vintf/manifest/android.hardware.gnss@2.1-service-qti.xml, use DEVICE_MANIFEST_FILE / DEVICE_MATRIX_FILE / vintf_compatibility_matrix / vintf_fragments instead!.`

Reply:
Oops... Forgot to force push the branches...
Will fix immediatelly.

Good eyes, Alex.

Reply:
Now, you are probably forgot to force-push vendor, because I can't see there 
`vendor/etc/vintf/manifest/vendor.qti.gnss@4.0-service.xml`